ws-nextjs-app/pages/[service]/send/[id]/types.ts

```diff
@@ -87,6 +88,7 @@ export type FieldData = {
   htmlType: HtmlType;
   messageCode: InvalidMessageCodes | null;
   wasInvalid: boolean;
+  wordLimit: number | undefined;
```

Comment:
Small thing but this could be `wordLimit?:` The `?` can be used as shorthand for declaring it as undefined.

Reply:
Thanks - I still get confused with Types!